tests/unit/backend/wmg/fixtures/test_snapshot.py

```diff
@@ -83,11 +83,18 @@ def random_expression_summary_values(coords):
     return {
         "nnz": randint(size=len(coords), low=0, high=100),
         "sum": random(size=len(coords)) * 10,
+        "nnz_raw": randint(size=len(coords), low=0, high=100),
+        "sum_raw": random(size=len(coords)) * 10,
     }
 
 
 def all_ones_expression_summary_values(coords):
-    return {"nnz": np.ones(len(coords)), "sum": np.ones(len(coords))}
+    return {
```

Comment:
The cubes generated for the test snapshot do not reflect any aggregation of raw counts because the generators are currently just random sampling or setting values to all ones. Updating this to perform aggregation is difficult because the cell types being used are not real and we don't know the relationships between the fake cell types. 

I think a unit test needs to be added to test the aggregator function (in `aggregate.py`) separately, and that should be enough to ensure that the counts are aggregating properly.